pkg/apitype/events.go

```diff
@@ -76,6 +76,24 @@ type SummaryEvent struct {
 	ResourceChanges map[string]int `json:"resourceChanges"`
 }
 
+// DiffKind describes the kind of a particular property diff.
+type DiffKind string
+
+const (
+	// DiffAdd indicates that the property was added.
+	DiffAdd DiffKind = "add"
+	// DiffAddReplace indicates that the property was added and requires that the resource be replaced.
+	DiffAddReplace DiffKind = "add-replace"
+	// DiffDelete indicates that the property was deleted.
+	DiffDelete DiffKind = "delete"
+	// DiffDeleteReplace indicates that the property was added and requires that the resource be replaced.
```

Comment:
might be worth indicating how this differs from DiffAddReplace

Reply:
Ah, this comment is just wrong.